wizelineacademy/chatGPT-sprint-1023
# Feat/ghaar/capstone init

### Background

Trying to get started with the project led to stumbling with a few bumps that can be avoided to newcomers. (e.g. finding that `dotenv` package actual name is `python-dotenv`)

### Problem

- Capstone project scaffold doesn't indicate recommended python version unless reading the provided scripts.
- Requirements file is non-existent, thus finding the right package names to any dependencies could take valuable time.

### Repro steps

- Clone the repo bare-bones
- Try to install dependencies to run `capstone_project/main.py`
- Result: modules can't be found

### Solution proposal

- Define a python version in a file
- Create requirements file
- Add instructions to the README file

### Testing steps

- Clone the repository again in a new directory to simulate a fresh start.
- Check if the Python version file (.python-version) exists and specifies the correct Python version.
- Run pip install -r requirements.txt to install the dependencies from the newly created requirements file.
- Check if the `python-dotenv` package is correctly listed in the requirements file.
- Try running `capstone_project/main.py` again. The modules should now be found and the script should run without any errors.
- Check the README file for the new instructions on setting up the project. Follow these instructions in a new environment to ensure they are clear and correct.
